src/System.Text.JsonLab/System/Text/Json/JsonReader.cs

```diff
@@ -10,10 +10,14 @@ namespace System.Text.JsonLab
     {
         // We are using a ulong to represent our nested state, so we can only go 64 levels deep.
         private const int MaxDepth = sizeof(ulong) * 8;
+        private static byte[] s_nullBytes = new byte[4] { (byte)'n', (byte)'u', (byte)'l', (byte)'l' };
```

Comment:
These aren't needed. We already have them defined within JsonConstants.
https://github.com/dotnet/corefxlab/blob/defd7e3ce7104b2fdd44d8bf5c542e2696ddb569/src/System.Text.JsonLab/System/Text/Json/JsonConstants.cs#L39-L45

Reply:
Ah, missed that.